calcom/cal.com
# chore: Deactivate automatic link prefetching for `/blog` links

## What does this PR do?

Conditionally disables NextJS' `prefetch` feature on `<Link/>`s.

This becomes relevant also for the future `/docs` & `/developer` pages. I will create subsequent PRs to update the links.
```diff
<Link
  // NextJS thinks this is an internal link and tries to prefetch it (also on hover), which fails
  // TODO: Extract this into a constant to handle /docs & /developer as well
+  prefetch={!learnMore.href.startsWith("https://cal.com/blog")}>
```

See the demo of the issue [in a screenshare](https://share.cleanshot.com/sKNLFhCz).
![CleanShot 2023-03-15 at 16 21 07@2x](https://user-images.githubusercontent.com/18185649/225356966-7dd0ed0c-a0f8-4ad0-b806-0859352e0a36.png)

Fixes # (issue)

### Demo
[Preview environment](https://share.cleanshot.com/WxYFLKQ7)



**Environment**: Staging(main branch) / Production

## Type of change




## How should this be tested?



- [x] Test in preview environment (basically following [in a screenshare](https://share.cleanshot.com/sKNLFhCz))
  - Log in to app.[previewEnv].com
  - Hover over the links of the tips cards (clear the localStorage entries if you don't see them)
  - Observer the Network tab and verify that no .json requests are issued
-> [see demo](https://share.cleanshot.com/WxYFLKQ7)
## Checklist



- I haven't read the [contributing guide](https://github.com/calcom/cal.com/blob/main/CONTRIBUTING.md)
- My code doesn't follow the style guidelines of this project
- I haven't performed a self-review of my own code and corrected any misspellings
- I haven't commented my code, particularly in hard-to-understand areas
- I haven't checked if my PR needs changes to the documentation
- I haven't checked if my changes generate no new warnings
- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
